xunit-autolabeler-v2/ast_parser/core/polyglot_parser.py

```diff
@@ -98,11 +98,19 @@ def _get_region_tag_from_line(line: Tuple[int, str]) -> Tuple[int, str]:
         line_text: str = line[1]
 
         tag = constants.REGION_TAG_ONLY_REGEX.search(line_text).group(0)
+
         return (line_num + 1, tag)  # +1 = convert to 0-indexed
 
     with open(source_path, 'r') as file:
+        file_lines = file.readlines()
+
+        # Remove _EXCLUDE tags
+        file_lines = [line for line in file_lines
+                      if '[START_EXCLUDE' not in line
```

Comment:
Suggestion: add a test case for this if we don't have one.

Reply:
Good catch - added.